ts/Series/Sankey/SankeySeries.ts

```diff
@@ -335,8 +335,16 @@ class SankeySeries extends ColumnSeries {
     }
 
     public drawDataLabels(): void {
-        ColumnSeries.prototype.drawDataLabels.call(this, this.points);
-        ColumnSeries.prototype.drawDataLabels.call(this, this.nodes);
+        for (const link of this.points) {
+            const dlOptions = this.options.dataLabels;
```

Comment:
this const is the same for all the links, so can be defined outside the scope of the for loop.